migration.go

```diff
@@ -14,7 +14,7 @@ import (
 
 type MigrationRecord struct {
 	VersionId int64
-	TStamp    time.Time
+	TStamp    string
```

Comment:
I don't think we want to store `TStamp` as `string` from now on. It's a breaking change and it would break everyone already using goose.

Instead, we wanna make the `Unmarshaler` work for this type on your set-up (most probably MySQL related?).

This is the type in Postgres:
`tstamp     | timestamp without time zone |  default now()`